release.Jenkinsfile

```diff
@@ -163,31 +163,55 @@ try {
                     }
 
                     stage('Copy kyma-installer artifacts') {
-                        copyArtifacts projectName: 'kyma/kyma-installer-artifacts', 
+                        copyArtifacts projectName: 'kyma/kyma-installer-artifacts',
                             selector: specific("${kymaInstallerArtifactsBuild.number}"),
                             target: "kyma-installer-artifacts"
                     }
 
                     // create release on github
                     stage("Publish ${isRelease ? 'Release' : 'Prerelease'} ${appVersion}") {
-                        
+
                         withCredentials(
-                                [string(credentialsId: 'public-github-token', variable: 'token')]
-                        ) { 
+                                [string(credentialsId: 'public-github-token', variable: 'token'),
+                                sshUserPrivateKey(credentialsId: "bitbucket-rw", keyFileVariable: 'sshfile')
+                        ]) {
+                            // Build changelog generator
+                            dir(changelogGeneratorPath) {
+                                sh "docker build -t changelog-generator ."
+                            }
+
+                            // Setup path to repository config file for `lerna-changelog`
+                            configFile = "./tools/changelog-generator/package.json"
```

Comment:
Use `def`
https://stackoverflow.com/questions/184002/groovy-whats-the-purpose-of-def-in-def-x-0